src/components/connection-prompt/ConnectSameDialog.svelte

```diff
@@ -12,32 +12,30 @@
   export let onConnectDifferentClick: () => void;
 </script>
 
-<main>
-  <div class="text-center mb-5 font-bold text-lg">
-    <h1>
-      {$t('connectMB.output.header')}
+<div class="text-center mb-5 font-bold text-lg">
```

Comment:
We don't use this dialog, so we might as well leave it as to avoid unnecessary conflict with upstream.